### PR TITLE
Fe feat/toggle save

### PR DESCRIPTION
**Pull request by jmartin777**
# Description

-CSS Styling with Toggle Save and Mushroom card.
-Toggle Save basic functionality implemented
-App rendermushroomcards renders togglesave outside of mushroom card

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] Testing
- [x] Refactoring

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -5,25 +5,29 @@ import ErrorPage from '../ErrorPage/ErrorPage';
...
+          <MushroomCard
             image={mushroom.image}
             latinName={mushroom.latinName}
             commonNames={mushroom.commonNames}
             probability={mushroom.probability}
-        />
-      </Link>
+          />
+        </NavLink>
+        <ToggleSave />
```
Comment (Elise-Jones) [inline src/Components/App/App.js:29]: Great work around handling the issue where togglesave automatically routed to the link.

File: src/Components/ToggleSave/ToggleSave.css [hunk 2]
```diff
@@ -1,15 +1,33 @@
...
-  }
-  
-  .toggled-on {
-    transform: rotate(20deg);
-  }
\ No newline at end of file
+  position: relative;
+  cursor: pointer;
+  display: inline-block;
+  z-index: 2;
```
Comment (Elise-Jones) [inline src/Components/ToggleSave/ToggleSave.css:5]: The power of z-indeX!

**Review by Elise-Jones — APPROVED** [review]

File: src/Components/App/App.js [hunk 3]
```diff
@@ -5,25 +5,29 @@ import ErrorPage from '../ErrorPage/ErrorPage';
 import FieldGuide from '../FieldGuide/FieldGuide';
 import './App.css';
 import Nav from '../Nav/Nav'
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
```
Comment (andreasorensen) [inline src/Components/App/App.js:8]: Good call changing the Link to NavLinks!

File: src/Components/App/App.js (hunk 1 above)
Comment (andreasorensen) [inline src/Components/App/App.js:29]: Glad you found a work around!

File: src/Components/ToggleSave/ToggleSave.css (hunk 2 above)
Comment (andreasorensen) [reply src/Components/ToggleSave/ToggleSave.css:5]: Do we still need the z index, with the changes you implemented?

**Review by andreasorensen — APPROVED** [review]

**Pull request merged**